tests/markdown/inlineCode/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -26,9 +26,9 @@ proseWrap: "always"
 
 \`\` 12\`34 \`\`
 
-\`\` \`12 \`\`
+\`\`  \`12 \`\`
```

Comment:
Not sure about this one. I think this output is technically more correct given the input as long as the goal is to _just_ add the gap between the ``` `` ``` and the content, which the code sort of implies.

I think it's easier to see what's going on here without the escaping so:

```md

==== input ====

`` `12``

==== output ====

`` `12 ``
```

Essentially before the output would have truncated the space to the left of the inner backtick; now it'll be preserved.

Reply:
This extra space looks strange and doesn't affect the output, so I think it shouldn't be added.

Reply:
Maybe it's because this hack, https://github.com/saramarcondes/prettier/blob/b761b16194731729ea62eebf1139be806c805bbf/src/language-markdown/utils.js#L193

Reply:
These [nodes are `InlineCode` though](https://astexplorer.net/)... am I missing where that function is used for inline code @fisker? I only see it used for `Code` nodes.

Reply:
Sorry, not related.

Reply:
@thorn0 or @fisker do you have any suggestions for how to deal with this? It does actually change the output from an AST perspective: https://spec.commonmark.org/dingus/?text=%60%60%20%6012%60%60%0A%0A%60%60%20%6012%20%60%60%0A%0A%60%60%20%20%6012%20%60%60

You can see that ``` `` `12`` ``` results in the value ``" `12"`` where as ``` `` `12 `` ``` results in ``"`12"``. With the current change, however, we preserve the AST's original value. This doesn't affect the rendering _in HTML_ because the leading whitespace inside the `<code>` element is ignored... but given that the formatter is changing the AST in this case I don't think it's safe to undo this change, even if we're assuming that the only output for Markdown rendering is HTML (not sure if that is the assumption here or not, but it's probably not a safe one? 🤔)

I'd like to keep this change the way it is to avoid changes to the AST's `value`.

Reply:
Is this expected? https://github.com/prettier/prettier/pull/8140/commits/1c6f7f888b5492a51e657a0c2c2e76f6081d3e09